pkg/transport/entry.go

```diff
@@ -47,20 +48,23 @@ type Entry struct {
 }
 
 // MakeEntry creates a new transport entry
-func MakeEntry(pk1, pk2 cipher.PubKey, tpType string, public bool, label Label) Entry {
-	return Entry{
-		ID:     MakeTransportID(pk1, pk2, tpType),
-		Edges:  SortEdges(pk1, pk2),
+func MakeEntry(initiator, target cipher.PubKey, tpType string, public bool, label Label) Entry {
+	entry := Entry{
+		ID:     MakeTransportID(initiator, target, tpType),
 		Type:   tpType,
 		Public: public,
 		Label:  label,
 	}
+	entry.Edges[0] = initiator
+	entry.Edges[1] = target
+	return entry
 }
 
 // SetEdges sets edges of Entry
-func (e *Entry) SetEdges(localPK, remotePK cipher.PubKey) {
-	e.ID = MakeTransportID(localPK, remotePK, e.Type)
-	e.Edges = SortEdges(localPK, remotePK)
+func (e *Entry) SetEdges(initiator, target cipher.PubKey) {
```

Comment:
Where are we using this function? I see it being used in tests but seems to be absent from codebase otherwise. Do we need it?

Reply:
Don't think we do, good catch